README.md

```diff
@@ -1,6 +1,8 @@
 # truckersmp-cli
 
-![CI](https://github.com/lhark/truckersmp-cli/workflows/CI/badge.svg)
+![Check main script](https://github.com/lhark/truckersmp-cli/workflows/Check%20main%20script/badge.svg)
+![Check proton.json](https://github.com/lhark/truckersmp-cli/workflows/Check%20proton.json/badge.svg)
+![Build/upload inject program](https://github.com/lhark/truckersmp-cli/workflows/Build/upload%20inject%20program/badge.svg)
```

Comment:
While these badges can somehow show if a project is in good shape I don't think they're particularly helpful either.
When you think they should be included then keep the important ones. Otherwise they get skipped past if they're too overwhelming anyway.

Reply:
Removed. Not really necessary, I think.